core/contract_test.go

```diff
@@ -0,0 +1,242 @@
+package core
+
+import (
+	_ "embed"
+	"encoding/json"
+	"testing"
+
+	"github.com/NethermindEth/juno/core/felt"
+)
+
+var (
+	//go:embed testdata/bytecode.json
+	bytecodeBytes []byte
+	//go:embed testdata/bytecode_genesis.json
+	bytecodeGenesisBytes []byte
+	//go:embed testdata/bytecode_0_8.json
+	bytecodeCairo08Bytes []byte
+)
+
+func TestClassHash(t *testing.T) {
+	var bytecode []*felt.Felt
```

Comment:
Can you confirm what Cairo version this is as well? We can also rename the `json` file to reflect the cairo version. 

Given that Starknet is known to sometimes change their hash implementations with newer Cairo versions, documenting the Cairo version could be useful in the future.

Reply:
Nice catch! Fixed now, thanks!